tests/hacker.test.js

```diff
@@ -114,9 +113,9 @@ describe("GET hacker", function() {
                 res.body.should.have.property("data");
 
                 let hacker = new Hacker(TeamHacker0);
-                chai.assert.equal(
-                    JSON.stringify(res.body.data),
-                    JSON.stringify(hacker.toJSON())
+                chai.assert.deepStrictEqual(
+                    res.body.data,
+                    JSON.parse(JSON.stringify(hacker)),
```

Comment:
So true, I think this was the most elegant solution I could find at that point (since hacker is a complex object of complex types like ObjectId and unfortunate fields like _id). I hadn't realized the toJSON method of the Schemas were overridden which is why the method didn't work for me. Just pushed a fix